build/import/Packages.targets

```diff
@@ -29,7 +29,7 @@
     <PackageReference Update="RoslynTools.SignTool"                                                   Version="$(RoslynToolsSignToolVersion)" />
     <PackageReference Update="XliffTasks"                                                             Version="1.0.0-beta.20574.1" />
     <PackageReference Update="Microsoft.DiaSymReader.Pdb2Pdb"                                         Version="$(MicrosoftDiaSymReaderPdb2PdbVersion)" />
-    <PackageReference Update="Microsoft.Net.Compilers.Toolset"                                        Version="4.2.0-1.final" />
+    <PackageReference Update="Microsoft.Net.Compilers.Toolset"                                        Version="3.11.0" />
```

Comment:
Why did this version downgrade?

Reply:
Unsure, I missed that on the merge conflict. Apologies. It's been reverted